vignettes/DESCRIPTION

```diff
@@ -0,0 +1,33 @@
+Package: hydromad
```

Comment:
Why is this DESCRIPTION file required in the vignettes folder? It seems like it's just a duplicate from the root folder?